src/components/common/Chatbox.js

```diff
@@ -13,9 +13,10 @@ const { useItemChat, useMembers, useAvatar, useItemMemberships } = hooks;
 
 const Chatbox = ({ item }) => {
   const { data: chat, isLoading: isChatLoading } = useItemChat(item.id);
-  const { data: members, isLoading: isMembersLoading } = useMembers([
-    ...new Set(chat?.messages?.map(({ creator }) => creator)),
-  ]);
+  const { data: memberships } = useItemMemberships(item.id);
+  const { data: members, isLoading: isMembersLoading } = useMembers(
+    memberships?.map((m) => m.memberId)?.toArray() || [],
```

Comment:
Do all members that have a membership on the item necessarily need to be loaded for the item's chat? What if some member did not use the chat on this item?

Reply:
@dialexo This is a good question, before, we did only load the members that had posted a message. But with the addition of the _mentions_, you want to be able to mention **all** members that have access to the item, for example to get their attention. If you have a better way of getting all members having access to the item, I am open to suggestions, in the mean time this is the best solution I could come up with.